### PR TITLE
Kim/cloud eng2

### PR DESCRIPTION
**Pull request by kengel100**
L2 and L3, with L4 still to come

### CODE REVIEW

File: Cloud-Engineering/L3-Senior-Cloud-Engineer.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# L3: Senior Software Engineer
```
Comment (pawelpabich) [inline Cloud-Engineering/L3-Senior-Cloud-Engineer.md:1]: Software => Cloud

File: Cloud-Engineering/L3-Senior-Cloud-Engineer.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- I can design and implement infrastructure as code.
+- I am proficient with virtualization, containerization, and serverless technologies.
+- I make appropriate scaling, reliability, and maintenance trade-offs as they occur in practice.
+- I identify, qualify and reduce technical debt as appropriate to my team's priorities.
+- I have a solid understanding of .NET fundamentals and I contribute to the Octopus Cloud codebase efficiently. (Equivalent to [L2 Software Engineer](../Software-Engineering/L2-Software-Engineer.md))
+- I identify and raise potential problems with my team before getting blocked, I ask for help when needed, and I offer help when I can be of assistance.
+
+<details>
+<summary>Examples</summary>
+- I led a small improvement project for Octopus Cloud from pitch to production, collaborating with my team.
```
Comment (pawelpabich) [inline Cloud-Engineering/L3-Senior-Cloud-Engineer.md:23]: 👍

File: Cloud-Engineering/L2-Cloud-Engineer.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+- **Responsibility and direction needed**: Weekly supervision needed from their manager or more senior engineers.
+
+## 🦉 Domain expertise
+
+- I have a solid understanding of server operating systems, network, and security fundamentals.
+- I can design and implement infrastructure as code.
+- I am becoming proficient with virtualization, containerization, and serverless technologies.
+- I am familiar with implementing and supporting CI/CD pipelines.
+- I am proficient with one or more scripting languages.
+- I am developing a solid understanding of .NET fundamentals so that I can contribute to the Octopus Cloud codebase. (Equivalent to [L1 Junior Software Engineer](../Software-Engineering/L1-Junior-Software-Engineer.md))
```
Comment (pawelpabich) [inline Cloud-Engineering/L2-Cloud-Engineer.md:18]: X Cloud Eng = Skills + X - 1 Software Engineer 👍

**Review by pawelpabich — COMMENTED** [review]
Looks really good 👍.

File: Cloud-Engineering/L2-Cloud-Engineer.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- **Planning horizon**: Current and next cycle
+- **Impact radius**: Self plus some team members
+- **Evaluation**: Manager
+- **Responsibility and direction needed**: Weekly supervision needed from their manager or more senior engineers.
+
+## 🦉 Domain expertise
+
+- I have a solid understanding of server operating systems, network, and security fundamentals.
+- I can design and implement infrastructure as code.
+- I am becoming proficient with virtualization, containerization, and serverless technologies.
```
Comment (pawelpabich) [inline Cloud-Engineering/L2-Cloud-Engineer.md:15]: This needs to go from L2 and L3

File: Cloud-Engineering/L3-Senior-Cloud-Engineer.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+- **Impact radius**: Peers (2-5)
+- **Evaluation**: Manager
+- **Responsibility and direction needed**: Takes vertical slices of larger products from idea to production.
+
+## 🦉 Domain expertise
+
+- I competently demonstrate my lived experience of server operating systems, network, and security fundamentals.
+- I can confidently step outside my own comfort zone and adapt quickly to new technologies.
+- I can design and implement infrastructure as code.
+- I am proficient with virtualization, containerization, and serverless technologies.
```
Comment (pawelpabich) [inline Cloud-Engineering/L3-Senior-Cloud-Engineer.md:15]: Same as above

**Review by pawelpabich — APPROVED** [review]
LGTM

**Review by hogfish — APPROVED** [review]
LGTM

**Pull request merged**